.github/workflows/ci.yml

```diff
@@ -20,13 +20,9 @@ jobs:
           node-version: ${{ matrix.node }}
           cache: 'npm'
       - run: npm ci
-      - run: npm install -g @lhci/cli@0.8.x
       - run: npm run build
-      - run: npx playwright install  --with-deps
-      - name: E2E
-        run: npm run ci:e2e
       - name: Publish to Chromatic
-        if: ${{ matrix.os == 'ubuntu-latest' }}
+        if: ${{ matrix.os == 'ubuntu-latest' && matrix.node < 17 }}
```

Comment:
only want to publish to chromatic one time and chromatic isn't availble on node 18 from what i can tell.

Reply:
Yes but it also errors on 18 so i had to use just 16.